app/src/main/res/layout/bottom_sheet_custom.xml

```diff
@@ -0,0 +1,96 @@
+        android:layout_height="@dimen/extra_space_top" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/btnsheet_top_section_height"
+        android:background="@drawable/btn_sheet_back"
+        android:orientation="vertical">
+
+        <ImageView
+            android:id="@+id/img_arrow"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:background="@color/colorPrimary"
+            android:src="@drawable/ic_arrow_drop_up_white_24dp" />
+
+        <TextView
+            android:id="@+id/sheet_slide_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center_horizontal"
+            android:text="@string/show_guide_text"
+            android:textColor="@color/white" />
+
+    </LinearLayout>
+
+    <android.support.v4.widget.NestedScrollView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/white"
+        android:orientation="vertical">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:padding="@dimen/bottom_sheet_padding">
+
+            <Space
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/space_length_btm_sheet" />
+
+            <TextView
+                android:id="@+id/guide_title"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:textSize="@dimen/bottom_sheet_title_text"
+                android:textStyle="bold"
+                tools:text="@string/wave_generator" />
+
+            <TextView
+                android:id="@+id/custom_dialog_text"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/bottom_sheet_margin"
+                tools:text="@string/multimeter_dialog_text" />
+
+            <ImageView
+                android:id="@+id/custom_dialog_schematic"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/bottom_sheet_margin"
+                android:contentDescription="@string/schematic_desc"
+                android:src="@drawable/sin_wave_guide" />
+
+            <TextView
+                android:id="@+id/custom_dialog_desc"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/bottom_sheet_margin"
+                android:background="@color/white"
+                tools:text="@string/lux_meter_intro" />
```

Comment:
I'm not sure how `lux_meter_intro` and `multimeter_dialog_text` can join here. Are they for testing?

Reply:
yes they are for testing thats why used them with toolns.

Reply:
So, can't we remove them and put real content?

Reply:
Actually, I am putting the real content dynamically as I have to implement other instruments also so can't hard code the layout according to one instrument.

Reply:
Exactly! So can we please remove this hardcoded value?